CarpetX/param.ccl

```diff
@@ -69,18 +69,23 @@ CCTK_INT ncells_z "Number of grid cells"
 
 
 
-BOOLEAN periodic_x "Periodic"
+RESTRICTED:
+
+BOOLEAN periodic "Periodic boundary conditions"
```

Comment:
The comments on these parameters should be improved. They should explain under which conditions there is a periodic boundary in a particular direction.

Reply:
will do.

Reply:
done.